modin/engines/base/frame/data.py

```diff
@@ -1154,7 +1154,7 @@ def _map_reduce(self, axis, map_func, reduce_func=None):
         )
         return self._compute_map_reduce_metadata(axis, reduce_parts)
 
-    def _map(self, func, dtypes=None, validate_index=False, validate_columns=False):
+    def _map(self, func, dtypes=None):
```

Comment:
Why do we need to remove `validate_columns`? Based on the paper the number of columns can change after `map` operator applied.

Reply:
We have made some changes for the sake of performance since the paper was written. Now, `map` must output the same labels metadata as the input.

We have a new operator `explode` which does the same thing as `validate_columns`, but it doesn't require us to handle the metadata in a complex way in `map` for the common cases.

Reply:
Okay, I see. Please, remove the docstring for `validate_index` and we will merge the changes.